src/main/java/net/coderbot/iris/uniforms/CommonUniforms.java

```diff
@@ -259,22 +258,12 @@ public int getAsInt() {
 
 			ItemStack stack = client.player.getItemInHand(hand);
 
-			if (stack == ItemStack.EMPTY || stack == null) {
+			if (stack == ItemStack.EMPTY || stack == null || !(stack.getItem() instanceof IrisItemLightProvider)) {
```

Comment:
> || !(stack.getItem() instanceof IrisItemLightProvider)
This part of this if statement is useless, Items will always have a implementation of `IrisItemLightProvider` because of the mixin.

Reply:
Good point! Slipped my mind.